source/server/server_configuration_parser.cpp

```diff
@@ -237,11 +221,62 @@ std::string ServerConfigurationParser::read_keycert(const std::string& filename)
   return data;
 }
 
+std::string ServerConfigurationParser::parse_bind_address() const
+{
+  // Use default address if none is specified
+  std::string parsed_bind_address = kDefaultAddressPrefix;
+
+  auto it = config_file_.find(kAddressJsonKey);
+  if (it != config_file_.end()) {
+    try {
+      parsed_bind_address = it->get<std::string>();
```

Comment:
It should probably allow for an address like "::" or "::1" which are valid IP addresses on their own. The caller appends ":" + <port>, which assumes the address returned here already includes the brackets if needed before appending the colon and port to form the full "endpoint address".  I'd be inclined to have the caller actually do this. Add brackets if the address contains a colon but no brackets.

Reply:
For the sake of simplicity we will only check for non-empty string and documented recommended usage for the new "address" property. I have a follow-up task to document syntax and provide some examples.